test/unit/math/prim/prob/neg_binomial_2_test.cpp

```diff
@@ -63,7 +63,7 @@ TEST(ProbDistributionsNegBinomial2, error_check) {
   error_msg
       = "neg_binomial_2_rng: Location parameter "
         "divided by the precision parameter is "
-        "inf, but must be finite!";
+        "inf";
```

Comment:
We did not lose anything. In this PR the error message changed slightly, while keeping the meaning the same. Instead of changing the expected error message in these tests, I instead shortened the expected error message to just the part that did not change. This makes the tests less brittle - less likely to fail due to a change in code that is not actually an error.

I would argue that all tests expecting exact wording of an error message are quite brittle. I prefer tests just checking the exception type and if really really necessary a work or a few from the error message.

Reply:
Ah, I see. Yes I agree wrt to the test brittleness.